HIP-UserStore.Model/Events/UserActivityBaseEvent.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace PaderbornUniversity.SILab.Hip.UserStore.Model.Events
+{
+    public abstract class UserActivityBaseEvent : IUserActivityEvent
+    {
+        public int Id { get; set; }
```

Comment:
In the `PhotoUploaded` and `PhotoDeleted` events we have these two properties: `Id` and `UserId`. For me, `Id` feels a bit redundant because each user can only have one photo, so the photo is really identified by the `UserId`, and not by some other ID. Maybe we should only store the user ID?